examples/quads.c

```diff
@@ -0,0 +1,216 @@
+#define _POSIX_C_SOURCE 200112L
+#define _XOPEN_SOURCE 600
+#include <drm_fourcc.h>
+#include <GLES2/gl2.h>
+#include <getopt.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <time.h>
+#include <unistd.h>
+#include <wayland-server-core.h>
+#include <wlr/backend.h>
+#include <wlr/backend/session.h>
```

Comment:
This is unused as well.

Reply:
I removed the unused includes in `examples/rotation` as well.